it/src/main/java/org/corfudb/universe/node/server/CorfuServerParams.java

```diff
@@ -0,0 +1,57 @@
+package org.corfudb.universe.node.server;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Builder.Default;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.ToString;
+import org.corfudb.universe.node.Node.NodeParams;
+import org.corfudb.universe.node.Node.NodeType;
+import org.corfudb.universe.node.server.CorfuServer.Mode;
+import org.corfudb.universe.node.server.CorfuServer.Persistence;
+import org.slf4j.event.Level;
+
+import java.time.Duration;
+
+@Builder(builderMethodName = "serverParamsBuilder")
+@Getter
+@AllArgsConstructor
+@EqualsAndHashCode(exclude = {"streamLogDir", "logLevel", "stopTimeout"})
+@ToString
+public
+class CorfuServerParams implements NodeParams {
+    @Default
+    @NonNull
+    private final String streamLogDir = "/tmp/";
+    @Default
+    @NonNull
+    private final int port = 9000;
+    @Default
+    @NonNull
+    private final Mode mode = Mode.CLUSTER;
+    @Default
+    @NonNull
+    private final Persistence persistence = Persistence.DISK;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)